### DISCUSSION TITLE
Import localized date

### DISCUSSION THREAD

**Discussion by maxxer** [firefly-iii/firefly-iii#8127] [Firefly III] [answered]
Hi. I'm trying to import a CSV that contains a date in format `d D Y, H:i:s`, which unfortunately is localized in the Italian language. So January is shown as `gen` instead of the English `jan`.

Is it possible to import such dates?

thanks

**Comment by JC5** [comment, marked as answer]
Of course, just read the documentation. 😁 

https://docs.firefly-iii.org/data-importer/advanced/json/#date

**Reply by maxxer** [reply]
🤦 thankyou!

**Reply by JC5** [reply]
No problem, it's a very obscure feature 😁

**Comment by psychowood** [comment]
Came here to ask the same thing, it was a nice finding :)
Would you mind adding the locale prefix detail [here](https://github.com/firefly-iii/data-importer/blob/8fc964f191c372ac8799cafca61ad90a791b2391/resources/views/import/004-configure/index.twig#L294)? I can provide a PR if you want.

**Reply by JC5** [reply]
Good idea, feel free to submit a PR!

**Reply by JC5** [reply]
Done.

**Reply by psychowood** [reply]
Thanks,  sorry for the delay,  didn't have a chance to do it yet

**Reply by JC5** [reply]
Not a problem. I was working on the new release and it was an easy addition. Thanks for the suggestion!
